SETUP.md

```diff
@@ -49,20 +51,52 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
 
 ## OS-specific prerequisites
 
-### OS X Mojave / Mavericks / Yosemite / El Capitan / Sierra
+### OS X Catalina
 
+1. Choose shell. Starting in Catalina, [the default shell for new users is zsh](https://support.apple.com/en-us/HT208050). Most developers at Code.org are still using bash so that may be a smoother experience for now.
+    <details>
+      <summary>To use bash:</summary>
+
+      * Switch the default shell back to bash and disable the warning:
+        * `chsh -s /bin/bash`
+        * Add the following to `~/.bash_profile` or your desired shell configuration file:
+          ```
+          export BASH_SILENCE_DEPRECATION_WARNING=1
+          ```
+    </details>
+    <details>
+      <summary>To use zsh:</summary>
+        
+      * Setup git autocompletion
+        * Add the following to `~/.zshrc` or your desired shell configuration file:
+          ```
+          # git prompt
+          source ~/bin/oh-my-zsh/git-prompt.sh
+          GIT_PS1_SHOWCOLORHINTS=1
+          GIT_PS1_SHOWDIRTYSTATE=1
+          GIT_PS1_SHOWUNTRACKEDFILES=1
+          setopt PROMPT_SUBST ; PS1='%m:%~$(__git_ps1 " (%s)")\$ '
+           
+          # git completion
+          source /Library/Developer/CommandLineTools/usr/share/git-core/git-completion.zsh >/dev/null 2>&1
+           
+          # make git checkout not show remote branches
+          GIT_COMPLETION_CHECKOUT_NO_GUESS=1
+          autoload -Uz compinit && compinit
+          ```
+    </details>
 1. Install Homebrew: `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 1. Install Redis: `brew install redis`
 1. Run `brew install https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb enscript gs mysql@5.7 nvm imagemagick rbenv ruby-build coreutils sqlite parallel`
     <details>
-        <summary>Troubleshoot: <code>Formula.sha1</code> is disabled or <code>Error: pdftk: undefined method sha1' for #&lt;Class:...&gt;</code></summary>
+      <summary>Troubleshoot: pdftk errors</summary>
 
-        If it complains about `Formula.sha1` is disabled, removing https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb from the above command seems to not have serious side effects (it will cause `PDFMergerTest` to fail). It may be a new URL is needed in the dependency list, see https://leancrew.com/all-this/2017/01/pdftk/
+      * If it complains about pdftk, removing https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb from the above command seems to not have serious side effects (it will cause `PDFMergerTest` to fail). It may be a new URL is needed in the dependency list, see https://leancrew.com/all-this/2017/01/pdftk/
```

Comment:
I forgot to ask @mvkski before he left whether we're likely to keep using pdftk for a while, or if we maybe have a plan to move off of it. 

either way, the workaround here look good to me.